earth_enterprise/src/common/khInsetCoverage.h

```diff
@@ -177,6 +178,15 @@ class khInsetCoverage {
                   uint stepOutSize,
                   uint paddingSize);
 
+  uint64 GetSize() {
+    uint64 extentsVecSize = 0;
```

Comment:
Isn't there a GetObjectSize method for this?

Reply:
There is a method for calculating the size of objects in a vector as well as other collections. The difference here is that CacheSizeCalculations currently does not have access to khExtents, as well as other classes that I've implemented GetSize mthods for. I did not include them in CacheSizeCalculations to avoid dependency problems.